eng/MSBuild/ProjectStaging.targets

```diff
@@ -22,10 +16,17 @@
     <Error Condition="'$(MinMutationScore)' != 'n/a' AND ('$(MinMutationScore)' &lt; 50)" Text="MinMutationScore property must be >= 50 for normal stage." />
   </Target>
 
-  <!-- Amend the description based on stage -->
-  <PropertyGroup>
-    <Description Condition="'$(Stage)' == 'dev'">Experimental package. $(Description)</Description>
-    <Description Condition="'$(Stage)' == 'obsolete'">Obsolete Package. $(Description)</Description>
-  </PropertyGroup>
+  <!--
+   Produce errors if we don't have all the right property values for non-production stages.
+
+   Note, that Arcade resets $(_PreReleaseLabel) to "ci" for non-official builds, which breaks the validation.
+   See Arcade SDK/Version.BeforeCommonTargets.targets for more details.
+    -->
+  <Target Name="_ValidateVersion" AfterTargets="GenerateNuspec" Condition=" '$(OfficialBuild)' == 'true' and '$(_IsStable)' != 'true' ">
```

Comment:
Where is `OfficialBuild` set? And what is the difference between that and `OfficialBuildId` which is the common one that gets set?

Reply:
It is being set by the Arcade SDK [here](https://github.com/dotnet/arcade/blob/1560b3406a8ebf9e267f7a1dddbf5238dd378081/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L9-L12):
```xml
  <PropertyGroup>
    <OfficialBuild>false</OfficialBuild>
    <OfficialBuild Condition="'$(OfficialBuildId)' != ''">true</OfficialBuild>
  </PropertyGroup>
```